lib/rspec/core/option_parser.rb

```diff
@@ -70,7 +70,18 @@ def parser(options)
           bisect_and_exit(argument)
         end
 
-        parser.on('--[no-]fail-fast', 'Abort the run on first failure.') do |value|
+        parser.on('--[no-]fail-fast[=COUNT]', 'Abort the run after a certain number of failures (1 by default).') do |argument|
```

Comment:
Changing this string to `'--[no-]fail-fast [count]'` drops the need for the `=`, I double checked.